src/style.css

```diff
@@ -1 +1,127 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 600;
+}
+
+:root {
+  --main-color: #3d4e61;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  color: var(--main-color);
+  font-weight: 300;
+}
+
+body {
+  /* padding: 0; */
+  box-sizing: border-box;
+}
+
+.search {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+  box-sizing: border-box;
+}
+
+.search--big {
+  width: 100%;
+  height: 70px;
+}
+
+.search--small {
+  width: 100%;
+  height: 42px;
```

Comment:
Avoid fixing container sizes unless specifically required by the task. Fixed sizes can lead to overflow issues or unintended scrollbars. It's often better to let content dictate the size of its container.